templates/shared/forms/interactive/storage.html

```diff
@@ -244,23 +31,29 @@ <h3 class="p-heading--5">How should we get in touch?</h3>
               </li>
               <li class="p-list__item">
                 <label for="email">Work email:</label>
-                <input required id="email" name="email" maxlength="255" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" required="required" />
+                <input required id="email" name="email" maxlength="255" type="email"
+                  pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" required="required" />
               </li>
               <li class="p-list__item">
                 <label for="phone">Mobile/cell phone number:</label>
                 <input required id="phone" name="phone" maxlength="255" type="tel" required="required" />
               </li>
               <li class="p-list__item">
                 <label class="p-checkbox">
-                  <input class="p-checkbox__input" value="yes" aria-labelledby="canonicalUpdatesOptIn" name="canonicalUpdatesOptIn" type="checkbox">
-                  <span class="p-checkbox__label" id="canonicalUpdatesOptIn">I agree to receive information about Canonical's products and services.</span>
+                  <input class="p-checkbox__input" value="yes" aria-labelledby="canonicalUpdatesOptIn"
+                    name="canonicalUpdatesOptIn" type="checkbox">
+                  <span class="p-checkbox__label" id="canonicalUpdatesOptIn">I agree to receive information about
+                    Canonical's products and services.</span>
                 </label>
               </li>
-              <li class="p-list__item">In submitting this form, I confirm that I have read and agree to <a href="/legal/data-privacy/contact">Canonical's Privacy Notice</a> and <a href="/legal/data-privacy">Privacy Policy</a>.</li>
+              <li class="p-list__item">In submitting this form, I confirm that I have read and agree to <a
+                  href="/legal/data-privacy/contact">Canonical's Privacy Notice</a> and <a
+                  href="/legal/data-privacy">Privacy Policy</a>.</li>
```

Comment:
This, too, seems slightly different from the copydoc. If it's good like this for @Sophie-32, you can disregard this comment.

Reply:
Instead of 'In submitting' should be 'By submitting'